vendor/assets/stylesheets/schedule/schedule.css

```diff
@@ -32,12 +32,23 @@ grid-day {
   float: left;
 }
 
+day-label {
+  color: #777777;
+  display:block;
+  margin: 0 auto;
+  text-align: center;
+  /* should be the same as grid-hour */
+  border-bottom: 1px solid #e7e7e7;
+  height:16px;
```

Comment:
They can, but unless the font size (and thus em, indirectly) is specified in terms of pixels, you can't get the exact render height of this element, and thus the offset needed to place an event in addEvent. Unless you wanted to go retrieve the rendered height of the element when doing that, but that's even more gross.